package.json

```diff
@@ -88,8 +88,6 @@
     "hammerjs": "^2.0.8",
     "highlight.js": "^11.11.1",
     "icomoon-free-npm": "^0.0.0",
-    "jquery": "^3.7.1",
```

Comment:
still downloaded by annotator, but it's already a win not to use it in our own code 🎉